Simulation/Source/Components/Collider.cpp

```diff
@@ -90,6 +90,21 @@ CollisionPoint Box::TestCollision(const Component::com_Transform* transformA, co
 	return CollisionAlgo::FindBoxPlane(this, transformA, colliderB, transformB);
 }
 
+void Box::SetInertia(float mass)
+{
+	float height = mMax.y - mMin.y;
+	float width = mMax.x - mMin.x;
+	float depth = mMax.z - mMin.z;
```

Comment:
Minimum vector will almost always be negative, I think. You may want to try Max + Min, rather than Max - Min